### ISSUE TITLE
Support 3D Models with Bone more than 256

### ISSUE THREAD

**Issue by TheOmnisimush** [FlaxEngine/FlaxEngine#1434]
**What happened :**
I have some FBX Models, intended for AAA quality games for Desktop and Console.
These models are correctly imported inside both Unity and Unreal, 
however when I tried to import to Flax as SkinnedModel, it gave me this error :
[Error] Cannot import model file. Imported model skeleton has too many bones. Imported: 333, maximum supported: 256. Please optimize your asset.

That's how I know that Flax has a limitation of 256 bones.

**What was expected :**
256 bone limitations should be increased to 4096, or removed completely so that Flax can support infinite Bones.

**Flax version:**
1.6.6334

**Comment by mafiesto4** [comment]
We are going to refactor mesh data to support selective streams thus allowing mesh to contain more custom data (eg. secondary texture coords or maybe more blend indices memory).

**Comment by HydrogenC** [comment]
Agree, model using bone physics also requires a large number of bones.

**Comment by mafiesto4** [comment]
Added in Flax 1.10. https://docs.flaxengine.com/manual/release-notes/1_10/index.html#new-meshes-vertex-format
